content/github/setting-up-and-managing-your-github-user-account/changing-your-primary-email-address.md

```diff
@@ -7,6 +7,11 @@ versions:
   free-pro-team: '*'
   enterprise-server: '*'
 ---
+{% note %}
+
+**Note:** Please note that you cannot change your *primary* email address to an email that is already set to be your *backup* email address.
```

Comment:
```suggestion
**Note:** You cannot change your primary email address to an email that is already set to be your backup email address.
```
If we remove the "Please note" it'll be even clearer for our translators ✨ I removed the italics (some of our older docs have them), but we don't generally add visual emphasis inside a paragraph.